charts/gardener-extension-provider-aws/templates/storageclass.yaml

```diff
@@ -9,4 +9,7 @@ provisioner: {{ .Values.config.etcd.storage.provisioner }}
 allowVolumeExpansion: true
 parameters:
   type: gp2
+{{- if .Values.config.etcd.storage.encrypted }}
+  encrypted: "true"
```

Comment:
How about:
```suggestion
  encrypted: {{ .Values.config.etcd.storage.encrypted }}
```

Reply:
The default value is `false` so it should not update the resource if not needed.